CodeStarterCamp_Week2/main.swift

```diff
@@ -1,12 +1,36 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week2
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-print("Hello, World!")
+// 로또 번호 생성 함수
+
+func makeLottoNums() -> Set<Int> {
+    var winningNumbers: Set<Int> = []
+    let candidates: [Int] = Array(1...45).shuffled()
+    
+    winningNumbers = Set(candidates.prefix(6))
+    
+    return winningNumbers
+}
+
+// 로또 번호 당첨 확인 함수
+
+func checkMyLotto(myNumbers: Set<Int>, realNumbers: Set<Int>) -> Void {
+    
+    let intersectionLotto: Set<Int> = myNumbers.intersection(realNumbers)
+    
+    guard intersectionLotto.count > 0 else {
```

Comment:
```suggestion
    guard intersectionLotto.count > .zero else {
```
취향 강요 2입니다 ㅎㅎ
(진짜 취향이라 안고치셔도 돼요 ㅎㅎ)